torch/onnx/__init__.py

```diff
@@ -196,9 +198,9 @@ def export(model, args, f, export_params=True, verbose=False, training=TrainingM
             Must be provided when exporting a ScriptModule or ScriptFunction, ignored otherwise.
             Used to determine the type and shape of the outputs without tracing the execution of
             the model. A single object is treated as equivalent to a tuple of one element.
-        strip_doc_string (bool, default True): do not include the field
-            ``doc_string``` from the exported model. Otherwise the field will mention the source
-            code locations for ``model``.
+        strip_doc_string (bool, default True): [Deprecated and ignored. Will be removed in next PyTorch release]
```

Comment:
Should we also mark `strip_doc_string=None` in export() in line 35 this file?

Reply:
No, it's not necessary.

Let's just keep the default value here as it is till we remove it in the release after this new release.